doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,262 @@
+.. _Ubuntu Service OS:
+
+Using Ubuntu as the Service OS
+##############################
+
+This document explains how to use Ubuntu as the Service OS with the ACRN hypervisor. This document
+builds on the :ref:`getting_started`. The Service OS used as a reference in Project ACRN is based
+on `Clear Linux`_. This document explains how to use Ubuntu instead, the User OS is the same one as described
+in the :ref:`getting_started`. The hardware platform used throughout this document is the `Intel NUC Kit
+NUC6CAYH <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc6cayh.html>`_.
+
+Install Ubuntu (natively)
+*************************
+
+Ubuntu 16.04.4 LTS was used throughout this document, other versions such as 18.04 may work too.
+
+* Download Ubuntu 16.04 from the `Ubuntu 16.04.4 LTS (Xenial Xerus) page
+  <https://www.ubuntu.com/download/desktop>`_ and select the `ubuntu-16.04.4-desktop-amd64.iso
+  <http://releases.ubuntu.com/16.04/ubuntu-16.04.4-desktop-amd64.iso>`_ image.
+
+* Follow Ubuntu's `online instructions <https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop?_ga=2.114179015.1954550575.1530817291-1278304647.1523530035>`_
+  to install it on your device.
+
+* While not strictly required, enabling SSH gives the user a very useful mechanism for accessing the
+  Service OS remotely or when running one or more User OS (UOS). Follow these steps to enable it on
+  the Ubuntu SOS:
+
+  .. code-block:: console
+
+     sudo apt-get install openssh-server
+     sudo service ssh status
+     sudo service ssh start
+
+.. note::
+   Make sure you have full internet access before proceeding to the next steps.
+
+Install ACRN
+************
+
+There is no Ubuntu package (``.deb`` or ``snap``) that can be used to easily install the ACRN components
```

Comment:
Maybe softer to say:

    ACRN components are distributed in source form, so you'll need to download
    the source code, build it, and install it on your device.

Reply:
Updated